a.yaml

```diff
@@ -0,0 +1,165 @@
+---apiVersion: v1
```

Comment:
Need a newline here.

Reply:
Oops, this was an old file that shouldn't have been committed. The golden file shows we don't generate this bad YAML.